api/apps/brand_partners/index.coffee

```diff
@@ -0,0 +1,11 @@
+express = require 'express'
+routes = require './routes'
+{ setUser, authenticated, adminOnly } = require '../users/routes'
+
+app = module.exports = express()
+
+app.get '/brand-partners', routes.index
```

Comment:
I think this should be underscore case to keep consistency with our main API & json fields.